tests/unit/modin/test_utils.py

```diff
@@ -841,44 +851,54 @@ def test_try_convert_to_simple_slice(input, output):
 
 
 def test_append_columns():
```

Comment:
Do we need to update an existing test, or could we just add a new test? Existing tests should still pass, correct?